prow/config/calico-config.yaml

```diff
@@ -0,0 +1,27 @@
+# This section includes base Calico installation configuration.
+# For more information, see: https://docs.tigera.io/calico/latest/reference/installation/api#operator.tigera.io/v1.Installation
+apiVersion: operator.tigera.io/v1
```

Comment:
Do they only offer an operator install...? Seems extremely overkill for a test

Reply:
They have a helm chart - we should use it.

Reply:
It's their documented mechanism even via Helm yep.

Reply:
they have a 'manifest' mode, seems better fit for us https://docs.tigera.io/calico/latest/getting-started/kubernetes/self-managed-onprem/onpremises#install-calico

Reply:
I also looked at the helm. I didn't see a great way to minimize the install with helm so for now going with the `manifest mode` ball of yaml method.